Repository searching examples (#1361)
PowerShell/PSResourceGet: Examples/InstallRepositorySearchingExamples.md

        ```
        
3) Installing with a package name specified and wildcard repository, eg: `Install-PSResource 'TestModule' -Repository *Gallery -PassThru`
    * When the package exists in both repositories:
        ```
        Name        Version Prerelease Repository
        ----        ------- ---------- ----------
        TestModule  1.0.0.0            PSGallery 
        ```
        Should install 'TestModule' from 'PSGallery'.
        
    * When the package exists in the first repository (PSSGallery), but not the second (NuGetGallery):
        ```
        Name        Version Prerelease Repository
        ----        ------- ---------- ----------
        TestModule  1.0.0.0            PSGallery 
        ```
        Should install 'TestModule' from 'PSGallery'.
        
    * When the package exists in the second repository (PSGallery), but not the first (NuGetGallery):
        ```
        Name        Version Prerelease Repository
        ----        ------- ---------- ----------
        TestModule  1.0.0.0            NuGetGallery 
        ```
        Should install 'TestModule' from 'NuGetGallery'.
        
    * When the package exists in neither repository:
        ```

>         Install-PSResource: Package 'TestModule' could not be found.

Reviewer: ```suggestion
        Install-PSResource: Package(s) 'TestModule' could not be installed from registered repositories 'PSGallery, NuGetGallery'.
```

        ```
        
4) Installing with a package name specified and multiple repository names specified, eg: `Install-PSResource 'TestModule' -Repository PSGallery, NuGetGallery -PassThru`

    * When the package exists in both repositories:
        ```
        Name        Version Prerelease Repository
        ----        ------- ---------- ----------
        TestModule  1.0.0.0            PSGallery 
        ```
        Should install 'TestModule' from 'PSGallery'.
        
    * When the package exists in the first repository (PSGallery), but not the second (NuGetGallery):
        ```
        Name        Version Prerelease Repository
        ----        ------- ---------- ----------
        TestModule  1.0.0.0            PSGallery 
        ```
        Should install 'TestModule' from 'PSGallery'.
        

>     * When the package exists the second repository (NuGetGallery), but not the first (PSGallery):

Reviewer: ```suggestion
    * When the package exists in the second repository (NuGetGallery), but not the first (PSGallery):
```

        ```
        Name        Version Prerelease Repository
        ----        ------- ---------- ----------
        TestModule  1.0.0.0            NuGetGallery 
        ```
        Should install 'TestModule' from 'NuGetGallery'.
        
    * When the package exists in neither repository:
        ```

>         Install-PSResource: Package 'TestModule' could not be found.

Reviewer: ```suggestion
        Install-PSResource: Package(s) 'TestModule' could not be installed from registered repositories 'PSGallery, NuGetGallery'.
```

        ```
        

> 5) Installing with a package name specified and both a repository name specified AND a repository name with a wildcard, eg: `Install-PSResource 'TestModule' -Repository *Gallery, LocalRepo`.

Reviewer: ```suggestion
5) Installing with a package name specified and both a repository name specified AND a repository name with a wildcard, eg: `Install-PSResource 'TestModule' -Repository *Gallery, otherRepository`.
```

    * This scenario is not supported due to the ambiguity that arises when a repository with a wildcard in its name is specified as well as a repository with a specific name. The command will display the following error:
        ```

>         Install-PSResource: Package 'TestModule' could not be found.

Reviewer: ```suggestion
        Install-PSResource: Repository name with wildcard is not allowed when another repository without wildcard is specified.
```
